slack_sdk/rtm/v2/__init__.py

```diff
@@ -100,14 +103,23 @@ def __init__(
         self.ping_pong_trace_enabled = ping_pong_trace_enabled
 
         self.message_queue = Queue()
-        self.message_listeners = []
+
+        def goodbye_listener(_self, event: dict):
```

Comment:
As an additional change, I've added goodbye event listener. I am doing the final test with this change.

```
2021-02-18 03:25:10.982 DEBUG connection.py (342): Received a new data frame (session id: 1315b2b2-29a9-4fc5-bb39-b155409acc99, opcode: text, payload: {"type": "goodbye", "source": "gateway_server"})
```